fcrepo-kernel-modeshape/src/test/java/org/fcrepo/kernel/modeshape/FedoraResourceImplTest.java

```diff
@@ -99,6 +103,18 @@
     private Node mockContainer;
 
     @Mock
+    private NodeType mockPrimaryNodeType;
+
+    @Mock
+    private NodeType mockMixinNodeType;
+
+    @Mock
+    private NodeType mockPrimarySuperNodeType;
+
+    @Mock
+    private NodeType mockMixinSuperNodeType;
```

Comment:
We have some kind of checkstyles thing that disallows multiple-field declarations. I think this is a good example of why we should change that.